src/render/stream.rs

```diff
@@ -56,8 +58,13 @@ async fn markdown_stream_inner(
     render: &mut MarkdownRender,
     abort: &AbortSignal,
     writer: &mut Stdout,
+    model_id: &Option<String>,
 ) -> Result<()> {
-    let mut buffer = String::new();
+    let mut buffer = {
+        let style = Style::new().fg(Color::Blue);
+        let model_id = model_id.clone().unwrap_or("Assistant".to_string());
+        style.paint(model_id + &String::from(") ")).to_string()
+    };
```

Comment:
I put `style` and `model_id` into a block intentionally because I wanted to clarify they are values not to used after L67.
If you feel this is not to easy to understand, please let me know. I'll fix it.